tools/challenge-helper-scripts/fs-utils.ts

```diff
@@ -24,6 +24,7 @@ export function getSuperBlockSubPath(superBlock: SuperBlocks): string {
     [SuperBlocks.FoundationalCSharp]: '19-foundational-c-sharp-with-microsoft',
     [SuperBlocks.UpcomingPython]: '20-upcoming-python',
     [SuperBlocks.A2English]: '21-a2-english-for-developers',
+    [SuperBlocks.RosettaCode]: '22-rosetta-code',
```

Comment:
Same here 😃 

It looks like this will need to be updated to something other than 22 since Tom is also using that the Python curriculum

https://github.com/freeCodeCamp/freeCodeCamp/pull/52628/files#diff-7dc5a3c37a34a8e58cef2f545da6da29e0b376364c3c16a85b4a46c83e7baf4f